telemetry-testing/src/test/java/io/opentelemetry/example/telemetry/ApplicationTest.java

```diff
@@ -74,16 +74,16 @@ public void testTelemetry() {
         .atMost(30, SECONDS)
         .untilAsserted(
             () -> {
-              var requests = collectorServer.retrieveRecordedRequests(request());
+              HttpRequest[] requests = collectorServer.retrieveRecordedRequests(request());
```

Comment:
Nit: were changes in this file required? The language level (11) should allow `var`